src/Orc.Theming/Models/IconInfo.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Orc.Theming;
+
+using System;
+
+public struct IconInfo
```

Comment:
This should be internal struct in the Win32/Structs folder with the Win32 namespace.